ros/src/tl_detector/site_traffic_light_config.yaml

```diff
@@ -4,15 +4,17 @@ camera_info:
   image_width: 800
   image_height: 600
 camera_preprocessing:
-  image_width: 300
-  image_height: 300
+  image_width: 512
+  image_height: 512
 stop_line_positions:
     - [8.0, 16.2]
 use_tensorflow_detection: True
-use_tensorflow_nth_frame: 6
-tensorflow_graph_path: 'light_classification/inference_graph/site/frozen_inference_graph.pb'
-tensorflow_confidence_thesh: 0.6
-detections_filter_size: 3
-no_detections_filter_size: 12
-no_detections_threshold: 0.8
-state_count_threshold: 2
+use_tensorflow_nth_frame: 2
+tensorflow_graph_path: 'light_classification/fcn_graph/checkpoint'
+tensorflow_confidence_thesh_green: 0.85
+tensorflow_confidence_thesh_yellow: 0.85
+tensorflow_confidence_thesh_red: 0.85
```

Comment:
sim config still uses the old single setting `tensorflow_confidence_thesh`, not split by RGB, (besides pointing to a .pb file instead of a directory).